packages/flutter/lib/fix_data.yaml

```diff
@@ -14,6 +14,453 @@
 
 version: 1
 transforms:
+  # Changes made in https://github.com/flutter/flutter/pull/94156
+  - title: "Migrate to 'ColorScheme.background'"
+    date: 2021-11-26
+    element:
+      uris: [ 'material.dart' ]
+      field: 'backgroundColor'
+      inClass: 'ThemeData'
+    changes:
+      - kind: 'rename'
+        newName: 'colorScheme.background'
+
+  # Changes made in https://github.com/flutter/flutter/pull/94156
```

Comment:
OR (more likely) this may be conflicting with a very similar fix below:

```yaml
# Changes made in https://github.com/flutter/flutter/pull/81336
  - title: "Migrate to 'ColorScheme.secondary'"
    date: 2021-04-30
    element:
      uris: [ 'material.dart' ]
      method: 'copyWith'
      inClass: 'ThemeData'
```

You may need to just add to that fix rather than having separate fixes.